libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -252,6 +249,43 @@ void WasmDialect::addEthereumExternals()
 	}
 }
 
+void WasmDialect::addDebugExternals()
+{
+	static vector<External> debugExternals {
+		{"print32", {c_i32}, {}},
+		{"print64", {c_i64}, {}},
+		{"printMem", {c_i32, c_i32}, {}},
+		{"printMemHex", {c_i32, c_i32}, {}},
+		{"printStorage", {c_i32}, {}},
+		{"printStorageHex", {c_i32}, {}},
+	};
+	for (External const& ext: debugExternals)
+		addBuiltinFunction("debug.", ext.name, ext.parameters, ext.returns, ext.controlFlowSideEffects);
+}
+
+BuiltinFunction& WasmDialect::addBuiltinFunction(
+	string const& _prefix,
+	string const& _name,
+	vector<std::string> const& _params,
+	vector<std::string> const& _returns,
+	ControlFlowSideEffects _sideEffects
+) {
+	YulString name{_prefix + _name};
```

Comment:
Is indentation off or is GH showing it weirdly?

Reply:
This is just GH.